shared/router-v2/left-tab-navigator.desktop.tsx

```diff
@@ -73,11 +76,17 @@ const LeftTabNavigator = React.memo(function LeftTabNavigator({
             />
           ))}
         </Kb.BoxGrow>
+        <ModalBackdrop hasModals={hasModals} />
```

Comment:
without this (as in master) you get this flicker as you transition in the wizard as the backdrop is connected to the individual modals. Instead we have one backdrop so we can have nice transitions and never flicker. I played with using filter/blur etc but it really hurts performance